.gitignore

```diff
@@ -0,0 +1,11 @@
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Go workspace file
+go.work
+
+# IDE related
+.idea
```

Comment:
Files/directories like `.idea` are specific for each developer’s environment. I think these should be ignored in the `~/.config/git/ignore` file, not in project’s `.gitignore`. The opposite approach would lead to flooding the `.gitignore` file with patterns for many editors and IDE in various configurations.

(Just to be clear, patterns like `*.test`, which are encountered by everyone regardless on their environment, are fine.)

Reply:
...and one more thing: these `.gitignore` rules are not specific to the `packets1` package, so I would rather put them in the root of the project.